Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -158,6 +158,7 @@ final class ProjectGenerator: ProjectGenerating {
                                     pbxproj: PBXProj) throws -> PBXProject {
         let defaultRegions = ["en", "Base"]
         let knownRegions = Set(defaultRegions + projectFileElements.knownRegions).sorted()
+        let attributes = ["ORGANIZATION": project.organization]
```

Comment:
that way this additional attribute is only added where a value is set, otherwise this would yield an empty `"ORGANIZATION"` to be set.